mmv1/products/dataform/Config.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'Config'
+description: |-
+  A singleton resource represents a Dataform configuration for project location.
+min_version: 'beta'
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/dataform/docs/'
+  api: 'https://cloud.google.com/dataform/reference/rest/v1beta1/projects.locations'
+
+id_format: 'projects/{{project}}/locations/{{region}}/config'
+base_url: 'projects/{{project}}/locations/{{region}}/config'
+create_url: 'projects/{{project}}/locations/{{region}}/config'
+create_verb: 'PATCH'
+update_url: 'projects/{{project}}/locations/{{region}}/config'
+update_verb: 'PATCH'
+self_link: 'projects/{{project}}/locations/{{region}}/config'
+update_mask: true
+import_format:
+  - 'projects/{{project}}/locations/{{region}}/config'
+exclude_delete: true
+custom_code:
+  test_check_destroy: templates/terraform/custom_check_destroy/skip_delete_during_test.go.tmpl
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+examples:
+  - name: 'dataform_config'
+    primary_resource_id: 'config'
+    min_version: 'beta'
+    vars:
+      key_ring_name: 'example-key-ring'
+      crypto_key_name: 'example-crypto-key-name'
+parameters:
+  - name: 'region'
+    type: String
+    required: true
+    min_version: 'beta'
+    description: 'A reference to the region'
+    immutable: true
+    url_param_only: true
+properties:
+  - name: 'name'
```

Comment:
This `name` attribute is a little weird. From what I see in the API documentation, it would just contain redundant information like project and region, so should it just be `output: true`? I'm also not sure why it would need a name since it is a singleton.